packages/quick-edit/package.json

```diff
@@ -23,6 +23,7 @@
 	"devDependencies": {
 		"@cloudflare/workers-tsconfig": "workspace:*",
 		"concurrently": "^8.2.2",
+		"quick-edit-extension": "workspace:*",
```

Comment:
This will cause changesets to bump the version of quick-edit when quick-edit-extension's version is bumped which will then cause it to be released with other packages/workers